notebooks/RSP_Zeropoint_Tutorial.ipynb

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    from lsst_science_pipeline import tap_query, list_of_calexp, warp_to_exposure
</pre><p>can you make this line assuming you have installed SLSim and you can state: from slsim.lsst_science_pipeline import ...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>